src/UI/Button/Button.tsx

```diff
@@ -0,0 +1,90 @@
+import style from "./Button.module.scss"
+import { Link } from 'react-router-dom';
+import { ReactNode } from 'react';
+
+import cnBind from 'classnames/bind'
+
+
```

Comment:
```suggestion
import cnBind from 'classnames/bind'

```

Пустая лишняя строка